.github/workflows/ci_cd.yml

```diff
@@ -179,7 +179,7 @@ jobs:
     name: "Deploy stable documentation"
     runs-on: ubuntu-latest
     needs: release
-    if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags') && !contains(github.ref, `a`) && !contains(github.ref, `b`)  && !contains(github.ref, `rc`) && !contains(github.ref, `post`)
```

Comment:
Think these need to be regular `'`?

Reply:
Indeed. Also at the moment it would never run, since 'ref/tags' include 'a'... Need to rework this